### PR TITLE
Add OrderBook and Ticker methods to Bybit Spot API

### PR DESCRIPTION
**Pull request by nuteater**
### Pull request checklist

- [x] Did you bump the project version?
- [x] Did you add a description to the Pull Request?
- [ ] Did you add new tests?
- [x] Did you add reviewers?

### CODE REVIEW

File: examples/Bybit/Spot.jl [hunk 1]
```diff
@@ -15,6 +15,10 @@ CryptoAPIs.Bybit.Spot.candle(;
 
 CryptoAPIs.Bybit.Spot.symbol_info()
 
+CryptoAPIs.Bybit.Spot.order_book(symbol = "ADAUSDT")
+
+CryptoAPIs.Bybit.Spot.ticker(symbol = "ADAUSDT")
+
```
Comment (dmitrii-doronin) [inline examples/Bybit/Spot.jl:21]: Kwargs must be always separated by a semicolon ';'

**Comment by dmitrii-doronin** [comment]
Hi! Thank you a lot for contributing. I've suggested a minor change to docs. Otherwise, the PR looks good to me!

File: src/Bybit/Spot/API/OrderBook.jl [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## Code samples:
+
+```julia
+using Serde
+using CryptoAPIs.Bybit
+
+result = Bybit.Spot.order_book(
+    symbol = "ADAUSDT",
+)
```
Comment (yshadrin) [inline src/Bybit/Spot/API/OrderBook.jl:51]: To continue about separating Args by `;`
```suggestion
result = Bybit.Spot.order_book(;
    symbol = "ADAUSDT",
)
```

**Comment by yshadrin** [comment]
Please don't forget to update minor in Project.toml. 
Thank you for your PR!

**Review by gryumov — APPROVED** [review]

**Pull request merged**